src/WebJobs.Extensions.DurableTask/EtwEventSource.cs

```diff
@@ -446,6 +447,18 @@ private EtwEventSource()
             this.WriteEvent(227, TaskHub, AppName, SlotName, FunctionName, InstanceId, OperationName, OperationId, FunctionType, ExtensionVersion, IsReplay);
         }
 
+        [Event(228, Level = EventLevel.Warning)]
+        public void TokenRenewalFailed(
+            string Resource,
+            string TenantId,
+            string ClientId,
+            long DelayMs,
+            string ExceptionMessage,
```

Comment:
Can we change this to `Details`? I'm trying to reduce the number of distinct columns we return from our Kusto queries to making searching through data easier. For other providers, we use `Details` to include exception information so I'd like to use that here too.
```suggestion
            string Details,
```

Reply:
Oh sure thing